arangod/Aql/ExecutionNode.cpp

```diff
@@ -1562,6 +1563,45 @@ void CalculationNode::toVelocyPackHelper(VPackBuilder& nodes, unsigned flags) co
   }
 
   nodes.add("expressionType", VPackValue(_expression->typeString()));
+  
+  if ((flags & SERIALIZE_DETAILS) &&
```

Comment:
I think I used SERIALIZE_DETAILS for stuff required for snippets. Mayve we should introduce a seperate flag for that?